client/web/compose/src/components/PageBlocks/Configurator.vue

```diff
@@ -478,53 +477,56 @@ export default {
       },
     },
 
-    autoCompleteSuggestionTree () {
+    autoCompleteParams () {
       const { fields = [] } = this.module || {}
       const moduleFields = fields.map(({ name }) => name)
+      const userProperties = this.$auth.user.properties() || []
+      const recordValueProperties = { value: 'values', properties: Object.keys(this.record.values) || [] }
```

Comment:
record can be undefined so you can move this to line 494 instead (otherwise this.record is undefined and throws an error)